backend/models.py

```diff
@@ -1,3 +1,66 @@
-# from django.db import models
+from django.db import models
 
 # Create your models here.
+
+
+class Faculty(models.Model):
+    name = models.CharField(max_length=150)
+
+
+class FieldOfStudy(models.Model):
+    faculty = models.ForeignKey(Faculty, on_delete=models.CASCADE)
+    name = models.CharField(max_length=150)
+    degree = models.IntegerField()
```

Comment:
Wydaje mi się, że rozważamy tylko studia pierwszego i drugiego stopnia, więc można zrobić to pole jako choice. Ewentualnie można rozważyć bardziej złożone przypadki tak jakie pierwszy stopień inżynierski a licencjacki oraz studia drugiego stopnia 1,5-roczne a 2-letnie

Reply:
Dobra, ma to sens, już poprawiam